### PR TITLE
Improve sc noise stability

### LINKED ISSUE #2920: SC: aesc_fsm_SUITE ==> all_tests.throughput.many_chs_msg_loop: FAILED
## Expected Behavior

Tests pass.

## Actual Behavior

Test case failing.

## Steps to Reproduce the Problem

Not reproduced yet.

## Logs, error output, etc.

https://circleci.com/gh/aeternity/aeternity/92359

### PR DESCRIPTION
**Pull request by velzevur**
While lurking for #2920, improve stability of noise session
Fixes #2920

### CODE REVIEW

File: apps/aechannel/src/aesc_session_noise.erl [hunk 1]
```diff
@@ -352,7 +352,7 @@ locate_fsm(Type, MInfo, #st{ init_op = {accept, SnInfo, _Opts} } = St) ->
     Info = Info0#{reestablish => (Type =:= ?CH_REESTABL)},
     Cands = get_cands(Type, Info),
     lager:debug("Cands = ~p", [Cands]),
-    try_cands(Cands, 3, 0, Type, Info, St).
+    try_cands(Cands, 5, 0, Type, Info, St).
```
Comment (tolbrino) [inline apps/aechannel/src/aesc_session_noise.erl]: The retry logic should not use a static backoff but at least an exponential backoff. It should also be configurable. Both improvements should be handled separately though.

**Comment by velzevur** [comment]
I got to a new error
```


=ERROR REPORT==== 22-Oct-2019::08:47:41 ===
Error in process <0.14095.0> on node aeternity_ct@localhost with exit value:
{timeout,[{aesc_fsm_SUITE,match_responder_and_initiator,2,
                          [{file,"/home/builder/aeternity/apps/aechannel/test/aesc_fsm_SUITE.erl"},
                           {line,2372}]},
          {aesc_fsm_SUITE,create_channel_from_spec,7,
                          [{file,"/home/builder/aeternity/apps/aechannel/test/aesc_fsm_SUITE.erl"},
                           {line,2268}]},
          {aesc_fsm_SUITE,create_multi_channel_,3,
                          [{file,"/home/builder/aeternity/apps/aechannel/test/aesc_fsm_SUITE.erl"},
                           {line,2221}]}]}

**Comment by velzevur** [comment]
Tests passed succesfully, restarted them

**Comment by velzevur** [comment]
Restarting again, just to make sure

**Comment by velzevur** [comment]
Restarting tests again for better confidence

**Pull request merged**